k8s/scripts/vars

```diff
@@ -38,4 +38,9 @@ TM_GENESIS_TIME='0001-01-01T00:00:00Z'
 
 # Blockchain ID must be unique for
 # every blockchain
-TM_CHAIN_ID='test-chain-rwcPML'
\ No newline at end of file
+TM_CHAIN_ID='test-chain-rwcPML'
+
+# IP Address of the resolver(DNS server).
+# i.e. CLUSTER-IP of `kubernetes` service(kube-dns)
```

Comment:
CLUSTER-IP or NODE-IP?

Reply:
`CLUSTER-IP` naming is a bit confusing but Kubernetes calls that IP `CLUSTER-IP`

Reply:
I will rename to `IP` to remove the confusion.